### ISSUE TITLE
I've downloaded the zip but there are no files?

### ISSUE THREAD

**Issue by trawra** [LinkedInLearning/learning-java-2825378#17]
I dont see any files in here... do I need to go to the b files separately?

**Comment by YuliaPrylypko** [comment]
Hi, maybe you need to change the branch from master to a specific lesson? This helped me.

**Comment by trawra** [comment]
Thank you, I'll try that!

I just wanted to download the whole thing via .zip.

**Comment by RaRe2604** [comment]
Yes @YuliaPrylypko. You are correct. We need change the branch to a **specific lesson.**
